src/chord.cpp

```diff
@@ -917,9 +917,6 @@ int Chord::ResetData(FunctorParams *functorParams)
 
 int Chord::PrepareDataInitialization(FunctorParams *functorParams)
 {
-    PrepareDataInitializationParams *params = vrv_params_cast<PrepareDataInitializationParams *>(functorParams);
-    assert(params);
-
```

Comment:
You also need to remove the `functorParams` argument name (also in the definition) because some compilers will complain about the argument not being used.

Reply:
As they're overrides Xcode complains about that. I'll drop this commit and leave the cleaning up to others.

Reply:
Weird. It should not and it is not for me with xcode Version 13.3.1 (13E500a). Which version are you using?

Reply:
Yes, the same version I got. Maybe I did something wrong? Anyway this PR was about the comparison fix in the first place.

Reply:
OK, I'll have a look